js/templates/trackerlist.es6.js

```diff
@@ -0,0 +1,17 @@
+const bel = require('./../../node_modules/bel');
+
+module.exports = function () {
+
+    var f = function(list) {
+        return list.map((site) => bel`<li> ${site.domain} = ${site.blocked} </li>`);
+    };
+
+    return bel`<div class="js-trackerlist js-menu-section">
+        <div class="js-menu-title">${this.model.heading}</div>
+        <ul class="js-menu-item-list">
+            ${f(this.model.testList)}
```

Comment:
This looks great. Only criticism is that I generally lean away from single variable function names in favor of something more descriptive. So maybe `f()` becomes `printList()` or something.